src/workerd/api/tests/webfs-test.js

```diff
@@ -125,7 +126,7 @@ export const asyncIteratorTest = {
 
     let count = 0;
     for await (const node of dir) {
-      ok(node.value instanceof FileSystemDirectoryHandle);
+      ok(node[1] instanceof FileSystemDirectoryHandle);
```

Comment:
Can you add a comment to what [0] and [1] refers to? It would be easier to debug if one of the tests fail in the future.